lib/r10k/forge/module_release.rb

```diff
@@ -89,7 +89,17 @@ def verify
       # @return [void]
       def unpack(target_dir)
         logger.debug1 "Unpacking #{download_path} to #{target_dir} (with tmpdir #{unpack_path})"
-        PuppetForge::Unpacker.unpack(download_path.to_s, target_dir.to_s, unpack_path.to_s)
+        file_lists = PuppetForge::Unpacker.unpack(download_path.to_s, target_dir.to_s, unpack_path.to_s)
+        logger.debug2 "Valid files unpacked: #{file_lists[:valid]}"
+        if !file_lists[:symlinks].empty?
+          logger.warn "Symlinks in modules are unsupported. These files were created as copies of the original files, "\
```

Comment:
I would rather that we either use string concatenation or just have line rather than escaping newlines, right now I'm not actually sure how this works but it seems like it's using some sort of implicit concatenation .